### PR TITLE
[G7T5-170] Lower Z-index of login button

### PR DESCRIPTION
**Pull request by quinncheong**
# Description
https://is212g7t5.atlassian.net/browse/G7T5-170

Login Button is no longer visible for modals.

image

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?
NA

# Checklist:

- [x] I have added the appropriate labels to my PR
- [x] My code follows the style guidelines of this project
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by theianchia — APPROVED** [review]
I feel like i rmb seeing the `z-50` being added for some reason but cant rmb what

**Comment by theianchia** [comment]
@jonathantan1425 this was your change right

File: app/src/layout/Login.jsx [hunk 1]
```diff
@@ -20,12 +25,10 @@ export default function Login() {
   }, []);
 
   return (
-    <Menu as='div' className='relative my-auto ml-5 z-50'>
-      <div>
-        <Menu.Button className='relative inline-flex items-center rounded-md border border-transparent bg-secondary px-4 py-2 text-sm font-medium text-white shadow-sm hover:bg-primary focus:outline-none focus:ring-2 focus:ring-accent3 focus:ring-offset-2 focus:ring-offset-gray-800'>
-          <span>{currentUserType || "Login"}</span>
-        </Menu.Button>
-      </div>
+    <Menu as='div' className='relative my-auto ml-5'>
```
Comment (jonathantan1425) [inline app/src/layout/Login.jsx:28]: Can we not entirely remove the `z-50` here? Perhaps lowering the value such that it still outweighs the modal?

image
Create job ([link](https://github.com/is212g7t5/is212g5t7-project/blob/1951cda4255ec79ebf6fe0f70fd170408c92baab/app/src/components/job/hr/CreateJob.jsx#L43)) seems to be blocking the user selection

**Review by jonathantan1425 — CHANGES_REQUESTED** [review]

File: app/src/layout/Login.jsx (hunk 1 above)
Comment (quinncheong) [reply app/src/layout/Login.jsx:28]: Oh man... Ok let me change it.

Comment (quinncheong) [reply app/src/layout/Login.jsx]: Changed the login z index to 10 and modal to 50.

**Review by jonathantan1425 — APPROVED** [review]

**Pull request merged**